tests/lib/util/glob-util.js

```diff
@@ -233,11 +223,12 @@ describe("globUtil", () => {
         it("should silently ignore default ignored files if not passed explicitly", () => {
```

Comment:
Is this test name still accurate given that we have an `assert.throws` in the test body?